.github/workflows/ci.yml

```diff
@@ -23,4 +30,11 @@ jobs:
       - run: yarn install --frozen-lockfile
       - run: npm run lint
       - run: npm run build
-      - run: npm test
+      - name: Run Tests on Node.js ${{ matrix.node-version }}
+        if: ${{ matrix.env.COVERAGE != '1' }}
+        run: npm test
+      - name: Run Tests w/Coverage on Node.js ${{ matrix.node-version }}
+        if: ${{ matrix.env.COVERAGE == '1' }}
```

Comment:
I think coverage can be enabled by default, any concerns?

Reply:
I guess I just didn't expect the outcome of running coverage for both versions of Node.js (redundant info).

Reply:
I see, coverage on one version should be enough.